ert3/config/_ensemble_config.py

```diff
@@ -1,5 +1,10 @@
 from typing import List, Optional
-from typing_extensions import Literal
+try:
```

Comment:
`typing_extensions` was not labeled as a requirement of ERT and CI started failing when only installing ERT and not `dev-requirements.txt`. As `Literal` is in `typing` from Python 3.8 I made a conditional import so that we only have to depend on `typing_extensions` for Python versions <= 3.7.